website/content/en/docs/best-practices/best-practices.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Operator Best Practices"
+linkTitle: "Best practices"
+weight: 1
+description: This guide describes the best practices concepts to write operators.
+---
+
+## Development
+
+Considerations for operator developers:
+
+- An Operator should manage a single type of application, essentially following the UNIX principle: do one thing and do it well.
+
+- If an application consists of multiple different tiers or components, multiple Operators should be written for each of them. If the application for example consists of Redis, AMQ and MySQL, there should be 3 Operators, not one.
+
+- If there is significant orchestration and sequencing involved, an Operator should be written that represents the entire stack, in turn delegating to other Operators for orchestrating their part of it.
```

Comment:
How would an operator delegate to another operator? Is this simply letting the other operator handle the reconciliation? This sentence is fine, but feels like we would have another item that talks about how the delegation would be handled.

Reply:
It came from https://github.com/operator-framework/community-operators/blob/master/docs/best-practices.md

I understand that it means that we would have for example Operator App and then another as DB. Then, we define that the APP depends on the DB. However, since it's very generic/conceptual I am not sure if we would able to add some concrete example.

Reply:
I think that is specified below either `An operator shouldn't deploy or manage other operators (such patterns are known as meta or super operators). It's the Operator Lifecycle Manager's job to manage the deployment and lifecycle of operators.` Note that we added a link for OLM. However, we can improve that in the future as well. Could that be?